### PR TITLE
fix: 🐛 MSDK-748 Handle CAId conversion in LinkCADocs

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### JIRA Link

https://polymath.atlassian.net/browse/MSDK-748

### CODE REVIEW

File: src/api/procedures/linkCaDocs.ts [hunk 1]
```diff
@@ -68,7 +72,7 @@ export async function prepareLinkCaDocs(
   }
 
   // eslint-disable-next-line @typescript-eslint/naming-convention
```
Comment (monitz87) [inline src/api/procedures/linkCaDocs.ts:74]: This can be deleted

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/api/procedures/linkCaDocs.ts (hunk 1 above)
Comment (prashantasdeveloper) [reply src/api/procedures/linkCaDocs.ts:74]: It was needed for `rawCAId`

```
74:9  error  Variable name `rawCAId` must match one of the following formats: strictCamelCase, UPPER_CASE  @typescript-eslint/naming-convention
```

Comment (monitz87) [reply src/api/procedures/linkCaDocs.ts:74]: Change the name to `rawCaId`

**Review by monitz87 — CHANGES_REQUESTED** [review]

**Review by monitz87 — APPROVED** [review]

**Review by polymath-eric — APPROVED** [review]

**Pull request merged**